src/raftstore/store/store.rs

```diff
@@ -372,17 +359,15 @@ impl<T, C> Store<T, C> {
         for region_id in self.region_ranges.values() {
             let region = self.region_peers[region_id].region();
             let start_key = keys::enc_start_key(region);
-            // TODO: use delete_range once #1250 is resolved.
-            try!(delete_file_in_range(
+            try!(util::delete_all_in_range(
```

Comment:
I don't think it's a good idea to call `delete_all_in_range` here. `delete_all_in_range` will seek first every time, this may make start process take a longer time.